src/clj/clojurians_log/slack_messages.clj

```diff
@@ -104,9 +154,139 @@
 
 (defn message->text
   "Convert Slack markup to plain text."
-  [message usernames]
+  [message usernames channel-names]
   (->> (-> message
            (mp/parse)
-           (replace-ids-names usernames))
+           (replace-user-ids-names usernames)
+           (replace-channel-ids-names channel-names))
        (map segment->text)
        (apply str)))
+
+
+(def test-response
```

Comment:
This doesn't seem to be used, it also looks a bit out of place here? Maybe move it to a `repl/` namespace?